src/components/App/App.component.jsx

```diff
@@ -1,55 +1,63 @@
-import React, { useLayoutEffect } from 'react';
+import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
+import Header from '../Header/Header.component';
+import SideNav from '../SideNav/SideNav.component';
+import LoginPage from '../../pages/Login/Login.page';
+import Home from '../../pages/Home/Home.component';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import Private from '../Private/Private.component';
+import Layout from '../Layout/Layout.component';
+import Video from '../../pages/Video/Video.component';
+import AuthProvider from '../../providers/Auth';
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
-
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
-
-    const intervalId = setInterval(rotateBackground, 3000);
-    body.addEventListener('click', rotateBackground);
-
-    return () => {
-      clearInterval(intervalId);
-      body.removeEventListener('click', rotateBackground);
-    };
-  }, []);
-
   return (
     <BrowserRouter>
       <AuthProvider>
-        <Layout>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            <Route exact path="/login">
-              <LoginPage />
-            </Route>
-            <Private exact path="/secret">
+        <Switch>
+          <Route exact path="/">
+            <Header />
+            <Layout>
+              <SideNav />
+              <Home />
+            </Layout>
+          </Route>
+          <Route exact path="/video/:id">
+            <Header />
+            <Layout>
+              <SideNav />
+              <Video />
+            </Layout>
+          </Route>
+          <Route exact path="/login">
+            <LoginPage />
+          </Route>
+          <Private exact path="/favorites">
+            <Header />
+            <Layout>
+              <SideNav />
+              <Home />
+            </Layout>
```

Comment:
Another way of doing this, and not repeat the `Header > Layout > SideNav` is putting the `Switch` inside of the `Layout`, next to the `SideNav`, in this way, the only thing that will change is the main content.